src/patternfly/components/ModalBox/styles.scss

```diff
@@ -2,85 +2,186 @@
   --pf-c-modal-box--BorderColor: transparent;
   --pf-c-modal-box--BorderSize: var(--pf-global--BorderWidth--sm);
   --pf-c-modal-box--BoxShadow: var(--pf-global--BoxShadow--lg);
+  --pf-c-modal-box--LineClamp: 3;
+  --pf-c-modal-box--MaxLineClamp: 5;
   --pf-c-modal-box--ZIndex: var(--pf-global--ZIndex--2xl);
   // MaxWidth is based on optimal line length for reading
   --pf-c-modal-box--MaxWidth: pf-size-prem(560px);
-  --pf-c-modal-box--MaxWidth--lg: pf-size-prem(1120px);
+  --pf-c-modal-box--lg--MaxWidth: pf-size-prem(1120px);
+
   // MinHeight is based on all the spacers we know we have, plus some LineHeight to make some space for at least one line of content in each section
   --pf-c-modal-box--MinHeight: calc(var(--pf-c-modal-box__header--PaddingTop) + var(--pf-c-modal-box__header--PaddingBottom) + var(--pf-c-modal-box__body--PaddingTop) + var(--pf-c-modal-box__body--PaddingBottom) + var(--pf-c-modal-box__footer--PaddingTop) + var(--pf-c-modal-box__footer--PaddingBottom) + calc(1rem * 4 * var(--pf-global--LineHeight--md)));
   // MaxHeight ensures that the modal will not go off the screen and instead the body will scroll
-  --pf-c-modal-box--MaxHeight: calc(100vh - (2 * (var(--pf-global--spacer--3xl))));
+  --pf-c-modal-box--MaxHeight: calc(100vh - var(--pf-global--spacer--2xl));
+  --pf-c-modal-box--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-modal-box--MarginLeft: var(--pf-c-modal-box--MarginRight);
+
+  // Box Header variables
+  --pf-c-modal-box__close--Top: var(--pf-global--spacer--sm);
+  --pf-c-modal-box__close--Right: var(--pf-global--spacer--md);
 
   // Box Header variables
   --pf-c-modal-box__header--PaddingTop: var(--pf-global--spacer--xl);
   --pf-c-modal-box__header--PaddingRight: var(--pf-global--spacer--xl);
   --pf-c-modal-box__header--PaddingBottom: var(--pf-global--spacer--sm);
   --pf-c-modal-box__header--PaddingLeft: var(--pf-global--spacer--xl);
 
+  // Box Header Title
+  --pf-c-modal-box__header-title--FontSize: var(--pf-c-title--m-3xl--FontSize);
+  --pf-c-modal-box__header-title--FontWeight: var(--pf-c-title--m-3xl--FontWeight);
+  --pf-c-modal-box__header-title--LineHeight: var(--pf-c-title--m-3xl--LineHeight);
+
   // Box Body variables
   --pf-c-modal-box__body--PaddingTop: var(--pf-global--spacer--sm);
   --pf-c-modal-box__body--PaddingRight: var(--pf-global--spacer--xl);
   --pf-c-modal-box__body--PaddingBottom: var(--pf-global--spacer--sm);
   --pf-c-modal-box__body--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-modal-box__body--MarginTop: var(--pf-global--spacer--xl);
 
   // Box Footer variables
   --pf-c-modal-box__footer--PaddingTop: var(--pf-global--spacer--xl);
   --pf-c-modal-box__footer--PaddingRight: var(--pf-global--spacer--xl);
   --pf-c-modal-box__footer--PaddingBottom: var(--pf-global--spacer--xl);
   --pf-c-modal-box__footer--PaddingLeft: var(--pf-global--spacer--xl);
 
+  // Box Footer Button variables
+  --pf-c-modal-box__footer--ButtonSpacer: var(--pf-global--spacer--md);
```

Comment:
why `ButtonSpacer` and not `MarginRight`?

Reply:
Good question. Based on mobile fitting when two buttons are present, ButtonSpacer would likely turn into a calc value that would determine margin-left AND margin-right values assuming button spacing will be tighter and or buttons will be centered at the smallest breakpoint. This hasn't been determined by design yet, but I think it will be when responsive views are refined. So, ButtonSpacer allows us to use a single, generic variable to manipulate margin-left AND margin-right values at various breakpoints rather than setting the values explicitly. Happy to change if you think it's better to set explicitly.